src/test/java/com/commercetools/sync/inventory/InventoryEntryMock.java

```diff
@@ -0,0 +1,135 @@
+package com.commercetools.sync.inventory;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.channels.ChannelRole;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.models.Reference;
+import io.sphere.sdk.types.CustomFields;
+import io.sphere.sdk.types.CustomFieldsDraft;
+import io.sphere.sdk.types.CustomFieldsDraftBuilder;
+import io.sphere.sdk.types.Type;
+
+import javax.annotation.Nonnull;
+import java.time.ZonedDateTime;
+import java.util.Map;
+
+import static java.util.Collections.singleton;
+import static java.util.Collections.singletonMap;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * Helper builder class for testing purpose. Allow to build mocks of {@link InventoryEntry}.
+ * Object are built using {@link org.mockito.Mockito#mock(Class)}. The getters methods of mocked object
+ * return values passed to builder.
+ */
+public class InventoryEntryMock {
```

Comment:
This is kind of mock utility, so the purpose of this class is the same as for "mockUtils": DRY and avoid test classes getting bigger. I need `InventoryEntry` mocks in different test classes so I provided customizable mock builder for myself instead of repeating slightly different mocking methods for every class. It's more convenient I think. I can move this to `InventoryMockUtils` as inner class to emphasize its purpose, how do you think?